src/map/map.h

```diff
@@ -553,6 +553,7 @@ enum status_point_types { //we better clean up this enum and change it name [Hem
 	SP_MOD_EXP=125,
 	SP_MOD_DROP=126,
 	SP_MOD_DEATH=127,
+	SP_BANKVAULT=128,
```

Comment:
The first 100 are used by client, all constant above 100 are not related to client/aegis.